runtime/lib/capi/ExecutionContext.hpp

```diff
@@ -182,7 +184,11 @@ class RTDevice {
         // Besides, this provides support for runtime device (RTD) libraries added to the system
         // path. This maintains backward compatibility for specifying a device using its name.
         // TODO: This support may need to be removed after updating the C++ unit tests.
-        if (rtd_lib == "lightning.qubit" || rtd_lib == "lightning.kokkos") {
+        if (rtd_lib == "null.qubit") {
+            rtd_name = "NullQubit";
+            _complete_dylib_os_extension(rtd_lib, "null_qubit");
+        }
+        else if (rtd_lib == "lightning.qubit" || rtd_lib == "lightning.kokkos") {
```

Comment:
Speaking off, lightning kokkos no longer lives here so it can be removed from the list right?

Reply:
removed